D1673/P1673.md

```diff
@@ -7754,7 +7720,7 @@ void matrix_vector_product(ExecutionPolicy&& exec,
                            OutVec y);
 ```
 
-[1]{.pnum} *Effects:* Computes $y$ such that $y = A x$.
+[1]{.pnum} *Effects:* Computes $y = A x$.
```

Comment:
Can we say this?  The issue is that in math font, the equals sign means equality.  There's no "assignment" in math.

Reply:
Right, but there's front matter that says how to read this.  First paragraph of [linalg.general] says that this means, "compute `$A x$`, and then assign it to the elements of function parameter `y`.  Any wording that uses math to describe the effects that is not this exact pattern (there are four instances of this) are worded a bit differently.